tests/load/test_job_client.py

```diff
@@ -772,18 +790,21 @@ def _load_something(_client: SqlJobClientBase, expected_rows: int) -> None:
         # 3 rows because we load to the same table
         _load_something(client, 3)
 
-        # adding new non null column will generate sync error
+        # adding new non null column will generate sync error, except for clickhouse, there it will work
         event_3_schema.tables["event_user"]["columns"]["mandatory_column"] = new_column(
             "mandatory_column", "text", nullable=False
         )
         client.schema._bump_version()
-        with pytest.raises(DatabaseException) as py_ex:
+        if destination_config.destination == "clickhouse":
```

Comment:
with the suggestion above, we should be able to have this run like all other destinations